README.md

```diff
@@ -0,0 +1,27 @@
+# AI-IaC-Prompts
+Give your feedback or thoughts on what prompts you would like to see improved by the Azure Deployments team.
+
+## Submitting a prompt
+Create a new issue by using the provided "feature request" template. Please ensure that you have checked for any duplicate issues before creating the issue. If you are inserting a prompt with sensitive data, make sure to redact any private info before creating your issue.
+
+### **IMPORTANT** What's a good prompt to submit to this forum? 
+For the purposes of this forum, a good prompt meets the following criteria:
+- Can be run in Copilot with a specific expected result
+- Specific resource types should be involved at some point contextually. This can be explicit (i.e. create a VM), or implicit (i.e. create a three-tiered application)
+- Expected result is generic enough to apply to other customers. While specific prompts are necessary for personal Copilot use, they will likely be unapplicable to others and thus deprioritized. Improving a generic variation of the specific prompt will improve the specific output.
```

Comment:
Why do we consider this "bad"? Looking at the example you gave below, it feels fair that a customer should be able to share feedback that their very specific scenario didn't work well:

> Deploy for me three VMs, two virtual networks, one AKS cluster, and five storage accounts under resource group "bad-example-rg"

Reply:
Agree with your assessment upon a second look. Do you have any alternative prompts that would match the idea of "non-applicable to other customers, too specific?"